crates/ironclaw_gateway/static/index.html

```
@@ -263,92 +375,274 @@ <h2 data-i18n="restart.title">Restart IronClaw Instance</h2>
+    </div>
+
+    <!-- Discover Tab (unified marketplace). The view owns its header. -->
+    <div class="tab-panel" id="tab-discover">
+      <div class="discover-container">
+        <div class="view-header">
+          <div class="view-header-title" data-i18n="discover.title">Discover</div>
+          <div class="view-header-desc" data-i18n="discover.subtitle">Channels, tools, and skills for your agent — or just ask it to build what's missing.</div>
+        </div>
+        <div class="discover-toolbar view-toolbar-sticky">
+          <div class="discover-search">
+            <input type="text" id="discover-search-input" data-i18n="discover.searchPlaceholder" data-i18n-attr="placeholder" placeholder="Search channels, tools, MCP servers, skills...">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)